### PR TITLE
Localization: Update local-map & Add tutorial for ndt localization module

### PR DESCRIPTION
**Pull request by tthhee**

### CODE REVIEW

File: modules/localization/conf/localization.conf [hunk 1]
```diff
@@ -211,3 +211,7 @@
 # need cpu to support avx intel intrinsics
 # default: false
 --if_use_avx=true
+
+--lidar_extrinsics_file=/apollo/modules/localization/msf/params/velodyne_params/velodyne64_novatel_extrinsics_example.yaml
+--map_dir=/apollo/Bag/Test_Data/Sunnyvale_Big_Loop
```
Comment (Capri2014) [inline modules/localization/conf/localization.conf]: This `map_dir` should be overwritten or mismatch with the ones we selected from dreamview frontend.

File: modules/localization/msf/local_map/base_map/base_map_matrix.cc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /******************************************************************************
- * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
```
Comment (Capri2014) [inline modules/localization/msf/local_map/base_map/base_map_matrix.cc]: 2019

File: modules/localization/conf/localization.conf (hunk 1 above)
Comment (jinghaomiao) [inline modules/localization/conf/localization.conf]: Please use the map directory gflag globally across all modules to minimize confusion

Comment (tthhee) [reply modules/localization/conf/localization.conf]: oh， sorry, it's my mistake.

**Pull request closed without merging**